tracer/src/Datadog.Trace.SourceGenerators/AspectsDefinitions/AspectsDefinitionsGenerator.cs

```diff
@@ -237,19 +240,22 @@ private static string GetSource(IReadOnlyList<string> definitions)
 using System.Linq;
 
 namespace Datadog.Trace.ClrProfiler
-{{
+{
     internal static partial class AspectDefinitions
-    {{
-        public static string[] Aspects = new string[] {{");
+    {
+        public static string[] Aspects = new string[] {
+""");
 
         foreach (var definition in definitions)
         {
-            sb.AppendLine(@$"@""{definition.Replace("\"", "\"\"")}"",");
+            sb.AppendLine(@$"""""""{definition}"""""",");
```

Comment:
errrrrr.... Is this right?! I _think_ you're trying to output a raw string literal here😅  But a raw string literal needs to have the start endings on different lines. So you should probably do something like this:
```suggestion
            sb.AppendLine(""""
            """
            """");
            ab.Append("            "); // need to make indent match
            sb.AppendLine(definition);
            sb.AppendLine(""""
            """
            """");
```

Also nit: don't use interpolation with string builder as you lose some of the benefit 😉

Reply:
You are right. Changed to good old escaped strings.